addons/storyshots/storyshots-core/src/frameworks/angular/loader.js

```diff
@@ -1,9 +1,15 @@
+import 'core-js';
+import 'core-js/es6/reflect';
+import 'core-js/es7/reflect';
 import hasDependency from '../hasDependency';
 import configure from '../configure';
 
 function setupAngularJestPreset() {
   // Needed to prevent "Zone.js has detected that ZoneAwarePromise `(window|global).Promise` has been overwritten."
+  require.requireActual('core-js');
   require.requireActual('core-js/modules/es6.promise');
+  // require.requireActual('core-js/es6/reflect');
```

Comment:
looks like you might be able to remove some of this now

Reply:
@ndelangen you changed that, can you take a look at this?

Reply:
Should remove if storyshots run successfully without it